src/core/storage/index.ts

```diff
@@ -5,7 +5,18 @@ export const LocalStorage = {
     await chrome?.storage?.local?.clear();
   },
   async set<TValue = unknown>(key: string, value: TValue) {
-    await chrome?.storage?.local?.set({ [key]: value });
+    try {
+      await chrome?.storage?.local?.set({ [key]: value });
+    } catch (e) {
+      const chromeError = chrome.runtime.lastError?.message;
+      logger.error(new RainbowError('LocalStorage write error'), {
+        message:
+          e instanceof Error
+            ? e.message
+            : `Unknown error: ${JSON.stringify(e)}`,
+        extra: { chromeError },
+      });
+    }
```

Comment:
need to see this so I can tell where https://rainbow-me.sentry.io/issues/6229126074/?environment=production&project=4504012577112064&query=is:unresolved%20issue.priority:%5Bhigh,%20medium%5D&stream_index=2 is coming from